setup.py

```diff
@@ -25,9 +25,14 @@ def find_version(*file_paths):
 requires = [
     'jmespath>=0.7.1,<2.0.0',
     'python-dateutil>=2.1,<3.0.0',
-    'urllib3>=1.25.4,<1.27',
+    'urllib3>=1.25.4,<1.27 ; python_version < "3.10"',
+    'urllib3>=1.25.4,<2.1 ; python_version >= "3.10"',
 ]
 
+extras_require = {
+    'crt': ['awscrt==0.16.26'],
+}
+
```

Comment:
Adding the environment markers to setup.cfg doesn't work with extras for older versions of setuptools. I've replicated the extras info here (similar to how we have the requires info in setup.cfg) to make sure we retain compatibility in those older setuptools versions.